openhtf/io/http_api.py

```diff
@@ -119,14 +121,25 @@ def do_GET(self):  # pylint: disable=invalid-name
       """
       result = {'test': util.ConvertToBaseTypes(self.executor.GetState()),
                 'framework': util.ConvertToBaseTypes(self.executor)}
-      self.wfile.write(json.dumps(result))
+      with open(os.devnull, 'w') as f:
+        with contextlib.redirect_stderr(f):  # Don't spam the terminal.
```

Comment:
What is being spammed to the terminal? Using `redirect_stderr` isn't recommended for libraries since it affects the global sys.stderr object.